experimental/PropertyDDS/packages/property-proxy/src/test/tsconfig.json

```diff
@@ -1,11 +1,9 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../../../../common/build/build-common/tsconfig.test.node16.json",
 	"compilerOptions": {
 		"rootDir": "./",
-		"outDir": "../../dist/test",
+		"outDir": "../../lib/test",
 		"types": ["node", "jest"],
-		"declaration": false,
-		"declarationMap": false,
 		"allowJs": true,
```

Comment:
If there are .js files in use, those are unlikely to be compatible in both CJS and ESM. Are there any? Are things working or not?

Reply:
I tested setting this to false and deleting this line. both are able to build, build tests and pass tests.

Reply:
But are there any .js files? We can't really rely on tests. There might not be coverage.

Reply:
there are no js files

Reply:
Based on that info, I think it's safe to remove that line from the config.